README.md

```diff
@@ -85,18 +116,71 @@ https://github.com/uggla/cloud_native_app
 
 ## Deliverables
 
- * On Gihub:
-   * Pipeline design documents.
+ * On Github:
+   * Pipeline design documents and code.
    * Application code.
-   * Heat template/ansible playbooks/scripts for Infra group.
+   * infrastructure code (Heat template/ansible playbooks/scripts).
 
  * Video that will present the CICD of the application in a new empty tenant.
 
-## Teachers support
+ * Your software factory should be ready to deliver a patch or a feature
+    requested by your customer.
+    This delivery will be in production in less than a day.
+    The feature or fix requested can come from the opensource project.
+
+*Note*: A complex feature or bug fix could be delivered several times,
+improving the application between each delivery. Your customer do not
+require this to be delivered just once. They prefer to see changes frequently.
+  But the overall application service must not be impacted by such update.
+
+  In the context of this training, no complex feature or bug fix will be
+  requested by your teachers.
+
+## Teachers interaction with students
+
+
+### Hardware
+
+Teachers will provide a remote access (though VPN) to :
+
+* the *customer production platform*
+    (2 servers pre-installed with a CentOS 7 distribution)
+    Your development team will have to setup them with an OpenStack as
+    requested by the customer
+    Your development team will deploy the customer application and application
+    health control on it.
+
+* a runnable *company Openstack tenant* to install your software factory.
```

Comment:
we need to introduce the "reference" term here somehow.

Reply:
ok. Added the *known* case to refer to what we use for us.